src/lib/ZeroWallet.ts

```diff
@@ -1,21 +1,37 @@
-import { ZeroWalletProviders } from "../types";
+import { ethers } from 'ethers';
+
+import {
+  GasTank,
+  GasTanks,
+  SupportedChainId,
+  ZeroWalletProviders,
+} from '../types';
 
 export class ZeroWallet {
+  #providers = {} as ZeroWalletProviders;
+  #biconomyDapps = {} as GasTanks;
```

Comment:
Can we abstract out the implementation of biconomy?

Biconomy is an implementation detail, and since we're looking at swapping it out I'd explicitly create an abstraction around biconomy. Consider using an adapter pattern or factory pattern for creating an instance of a gas tank that internally uses biconomy

Reply:
Done. Can you check?